default.json5

```diff
@@ -77,22 +77,64 @@
       ],
     },
     {
-      groupName: "all major dependencies",
-      groupSlug: "all-major",
+      matchManagers: [
```

Comment:
This changes more than just splitting npm dependencies to their own PRs, because previously all managers were matched (also dockerfile, github-actions).
@ymengesha what is your take on this?

Reply:
In addition to what Silvio said, now that we're touching the npm rules, we have to question whether we really need the semantic version grouping for npm dependencies. If the experience we have so far says sth different, then we can keep the original package rule as is and only add specific overriding ones for the npm manager. 

Otherwise, if we still wanna stick with the current configuration, then we probably need to introduce templates to keep our configs clean. e.g.

```
{
  groupName: "all major {{manager}} dependencies",
  groupSlug: "all-major-{{manager}}",
  matchUpdateTypes: [
    "major",
  ],
},
{
  groupName: "all minor {{manager}} dependencies",
  groupSlug: "all-minor-{{manager}}",
  matchUpdateTypes: [
    "minor",
  ],
},
{
  groupName: "all patch {{manager}} dependencies",
  groupSlug: "all-patch-{{manager}}",
  matchUpdateTypes: [
    "patch",
  ],
  matchCurrentVersion: "!/^0/",
  automerge: true,
  platformAutomerge: false,
},
```
Of course, this would mean we get separate PRs for the different managers.